resources/prosody-plugins/mod_muc_lobby_rooms.lua

```diff
@@ -149,7 +168,7 @@ function attach_lobby_room(room)
         -- which can leave the room with no occupants and it will be destroyed and we want to
         -- avoid lobby destroy while it is enabled
         new_room:set_persistent(true);
-        module:log("debug","Lobby room jid = %s created",lobby_room_jid);
+        module:log("info","Lobby room jid = %s created", lobby_room_jid);
```

Comment:
Do we want to log which user created it?  (Or would that be PII we don't want in the logs?)

Reply:
hum ... maybe we can print the muc jid ...

Reply:
Oh and there are cases where this is created from the backend :) so it will be null.